sql2.txt

```diff
@@ -17,29 +42,86 @@ may have ordered the same product in several different orders.  So the total qua
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
 
+              - SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity
+                FROM Customers c
+                JOIN Orders o
+                ON c.CustomerID = o.CustomerID
+                JOIN OrderDetails od
+                ON o.OrderID = od.OrderID
+                JOIN Products p
+                ON od.ProductID = p.ProductID
+                GROUP BY c.CustomerID, p.ProductID;
+
+
+
+
+
 
 
 4. Extend the previous query to give the total price as well as the customer name, product name, 
 and total quantity.  Here you will have
 (SUM(Quantity) * Price) AS TotalPrice.  Paste your SQL statement below.
 
+              - SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity, (SUM(od.Quantity) * p.Price) AS TotalPrice
+                FROM Customers c
+                JOIN Orders o
+                ON c.CustomerID = o.CustomerID
+                JOIN OrderDetails od
+                ON o.OrderID = od.OrderID
+                JOIN Products p
+                ON od.ProductID = p.ProductID
+                GROUP BY c.CustomerID, p.ProductID;
+
+
+
+
```

Comment:
noice:
when formatting for clarity - (easier to read in the future):

`SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity, (SUM(od.Quantity) * p.Price) AS TotalPrice
FROM Customers c
JOIN Orders o ON c.CustomerID = o.CustomerID
JOIN OrderDetails od ON o.OrderID = od.OrderID
JOIN Products p ON od.ProductID = p.ProductID
GROUP BY c.CustomerName, p.ProductName, p.Price;`